api/client/samples/analogous_years/lib/final_ranks_computation.py

```diff
@@ -47,7 +47,7 @@ def enso_data(start_date):
                         'item_id': 13495,
                         'region_id': 0,
                         'source_id': 124,
-                        'start_date': start_date,
+                        'start_date_bound': start_date,
```

Comment:
definitely unhelpful to change thi svariable name :)    this object is now incompatible with the thing we call "data_series" everywhere.  treat "data_series" as  a  subclass of "dict" ,, i.e. the fields are properties with standard meaning

Reply:
Okay, I will change it here.